components/web/firefox/patches/33-js-gc-memory.patch

```diff
@@ -0,0 +1,20 @@
+--- firefox-52.9.0esr/js/src/gc/Memory.cpp.orig	2018-09-26 00:23:36.691341198 +0000
++++ firefox-52.9.0esr/js/src/gc/Memory.cpp	2018-09-26 00:25:20.735665686 +0000
+@@ -501,7 +501,7 @@
+ MapMemoryAt(void* desired, size_t length, int prot = PROT_READ | PROT_WRITE,
+             int flags = MAP_PRIVATE | MAP_ANON, int fd = -1, off_t offset = 0)
+ {
+-#if defined(__ia64__) || (defined(__sparc64__) && defined(__NetBSD__)) || defined(__aarch64__)
++#if defined(__ia64__) || (defined(__sparc64__) && defined(__NetBSD__)) || defined(__aarch64__ || (defined(__sun) && defined(__x86_64__)))
```

Comment:
Sorry, but this seems to be no-op , we build for 32-bit arch, __x86_64__ is not defined.

Reply:
That was not the point ;) I wanted to make sure that this patch is used by anybody willing to work on Firefox 64bit. This is not useful aymore since I've just built 60esr. I will update the PR with it.